apps/server/src/endpoints.ts

```diff
@@ -86,10 +88,17 @@ export const createGame =
       switch (body.modelType) {
         case ModelType.THREAT_DRAGON: {
           // TODO: validation
-          await gameServer.db.setModel(
-            matchID,
-            JSON.parse(body.model as string) as ThreatDragonModel,
-          );
+          logEvent(`printing body: ${body.model}`);
```

Comment:
This log statement should be removed. The model can potentially contain sensitive data that MUST NOT end up in the logs.